clang/test/Sema/warn-lifetime-safety-dataflow.cpp

```diff
@@ -59,6 +59,7 @@ int return_int_val() {
   int x = 10;
 // CHECK: Block B{{[0-9]+}}:
 // CHECK:   Issue ([[L_X:[0-9]+]] (Path: x), ToOrigin: {{[0-9]+}} (Expr: DeclRefExpr))
+// CHECK:   Expire (0 (Path: x))
```

Comment:
This should be `L_X`​ instead of 0.

Same below. We should avoid depending on the LoanID which can change due to arbitrary reasons.